dispatches/models/fossil_case/thermal_oil/thermal_oil.py

```diff
@@ -100,7 +101,7 @@ class _StateBlock(StateBlock):
     def initialize(self, state_args={}, state_vars_fixed=False,
                    hold_state=False, outlvl=idaeslog.NOTSET,
                    temperature_bounds=(260, 616),
-                   solver='ipopt', optarg={'tol': 1e-8}):
+                   solver=None, optarg={'tol': 1e-8}):
```

Comment:
We generally use `optarg=None` - the IDAES config has a set of default options which includes solver tolerance (1e-6?)